src/app/context/wallet/BTCWalletProvider.tsx

```diff
@@ -88,6 +93,9 @@ export const BTCWalletProvider = ({ children }: PropsWithChildren) => {
     setAddress("");
   }, []);
 
+  const [filterOrdinalsModalOpen, setFilterOrdinalsModalOpen] = useState(false);
+  const [shouldFilterOrdinals, setShouldFilterOrdinals] = useState(true);
```

Comment:
We need make wallet providers stateless, we moved all application's state to `/app/state` folder